src/DETHRACE/common/piping.c

```diff
@@ -75,7 +75,8 @@ int PipeSearchForwards() {
 int IsActionReplayAvailable() {
     LOG_TRACE("()");
 
-    return gPipe_buffer_start != NULL;
+    STUB_ONCE();
```

Comment:
did you mean to re-stub this?

Reply:
No, this is an accidental change.
Thanks for pointing it out.